specs/protocol/README.md

```diff
@@ -408,19 +408,19 @@ Currency amounts have type `DecimalString`, which is string containing a decimal
 ### `RFQ (Request For Quote)`
 > Alice -> PFI: "OK, that offering looks good. Give me a Quote against that Offering, and here is how much USD (payin currency) I want to trade for BTC (payout currency). Here are the credentials you're asking for, the payment method I intend to pay you USD with, and the payment method I expect you to pay me BTC in."
 
-| field        | data type                                        | required | description                                                 |
-| ------------ | ------------------------------------------------ | -------- | ----------------------------------------------------------- |
-| `offeringId` | string                                           | Y        | Offering which Alice would like to get a quote for          |
-| `claimsHash` | string[]                                         | Y        | Salted hash of the claims appearing in `privateData.claims` |
-| `payin`      | [`SelectedPayinMethod`](#selectedpaymentmethod)  | Y        | selected payin amount, method, and details                  |
-| `payout`     | [`SelectedPayoutMethod`](#selectedpaymentmethod) | Y        | selected payout method, and details                         |
+| field        | data type                                        | required | description                                                                                                                                                                                                                                                           |
+|--------------|--------------------------------------------------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `offeringId` | string                                           | Y        | Offering which Alice would like to get a quote for                                                                                                                                                                                                                    |
+| `claimsHash` | string                                           | N        | Salted hash of the list the claims appearing in `privateData.claims`. Omit `claimsHash` to denote that the RFQ contains no claims, i.e. `privateData.claims` is omitted. If `privateData.claims` is an empty list, `claimsHash` is the salted hash of the empty list. |
```

Comment:
why have a salted hash of a list as opposed to no value at all?

Reply:
I mention this in the description for `privateData.claims` that the RFQ creator should prefer to have null/no value rather than an empty list. But i also wanted to be explicit about how the empty list case should be handled.

Reply:
I would remove the possibility altogether

Reply:
Good call. Updated